migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+type: specs.openrewrite.org/v1beta/recipe
```

Comment:
Not sure how this aligns with the standard, but wouldn't it make sense to:

1. Attach the specific AF version these YAMLs act on, or
2. Move these YAMLs in a dedicated `4.7` directory

Reply:
If we wouldn't, I imagine file name clashes to occur.

Reply:
Renamed the files with a `-47` suffix, [similar to Spring](https://github.com/openrewrite/rewrite-spring/tree/main/src/main/resources/META-INF/rewrite).